homeassistant/components/renault/renault_coordinator.py

```diff
@@ -18,6 +19,9 @@
 
 T = TypeVar("T", bound=Optional[KamereonVehicleDataAttributes])
 
+# We have potentially 7 coordinators per vehicle
+_PARALLEL_SEMAPHORE = asyncio.Semaphore(1)
```

Comment:
This replaces the `PARALLEL_UPDATES` feature that I don't think works in this scenario.
But maybe `@Throttle` would work better?